packages/app/src/components/Admin/Customize/CustomizeLogoSetting.tsx

```diff
@@ -20,14 +20,14 @@ const CustomizeLogoSetting = (): JSX.Element => {
   const [isImageCropModalShow, setIsImageCropModalShow] = useState<boolean>(false);
   const [isDefaultLogo, setIsDefaultLogo] = useState<boolean>(true);
   const [retrieveError, setRetrieveError] = useState<string | null>(null);
-  const [currentBrandLogoSrc, setCurrentBrandLogoSrc] = useState< string | null >(null);
+  const [customizedLogoSrc, setCustomizedBrandLogoSrc] = useState< string | null >(null);
```

Comment:
The second returned value should be named `setCustomizedLogoSrc` instead of `setCustomizedBrandLogoSrc`